### PR TITLE
#162949227 Add a default pagination

### PR DESCRIPTION
**Pull request by kwanj-k**
#### What does this PR do?

Create a default pagination class of ten items per page

#### Description of Task to be completed?

* Add a default authentication class in settings.py

#### How should this be manually tested?

1. Clone the repo: `git clone https://github.com/andela/ah-the-unsullied.git`
2. Cd into the cloned repo 
3. Git checkout to branch ft-article-pagination-162949227
4. Set up a virtual environment and install requirements:
      ```
            python3 -m venv venv
            source venv/bin/activate
            pip3 install -r requirements.txt

     ```
5. Set up a .env file in the same level as the venv file and add the following:
      ```
            source venv/bin/activate
            export SECRET_KEY='yoursecretkey'
            export DATABASE_URL='postgres://yourdatabasename'
            export DJANGO_SETTINGS_MODULE='authors.settings'
            export CLOUDINARY_NAME=' '
            export CLOUDINARY_KEY=' '
            export CLOUDINARY_SECRET=' '

      ```
6. Then run: 
     ``` 
            source .env 
            ./manage.py migrate
            ./manage.py runserver
      ```

7. Fire up postman and test the following endpoints:

| EndPoint                                        | Functionality                           |
| -------------------------------- |:----------------------------:|
| POST   api/users/                          | Create user profile                 |
| GET     api/profiles/                       | List all user profiles                | 
                                                   
use the following payload format to create a profile on registration:
```
{
  "user": {
    "email": "jake@jake.jake",
    "username": "jake",
    "password":"password"
  }
}
```
Once the profiles are more than 10, the rest are taken to the next page.

#### What are the relevant pivotal tracker stories?
[#162949227](https://www.pivotaltracker.com/story/show/162949227)

#### Screenshots (if appropriate)
Get all profiles returns 10 profiles per page:
screenshot 2019-01-25 at 10 08 15

### CODE REVIEW

File: authors/tests/test_pagination.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from django.urls import reverse
+
+# local imports
+from .base_test import TestBase
+
+
+class TestPagination(TestBase):
+    """This class has tests for pagination setting."""
+
+    def get_token(self):
```
Comment (indungu) [inline authors/tests/test_pagination.py]: This method could serve better if it was defined in the `TestBase` class, don't you agree?

**Review by indungu — CHANGES_REQUESTED** [review]
To best keep our code base as DRY as possible please make the following change.

File: authors/tests/test_pagination.py (hunk 1 above)
Comment (kwanj-k) [reply authors/tests/test_pagination.py]: Yes

**Review by indungu — APPROVED** [review]

**Comment by indungu** [comment]
@kwanj-k please update the screenshot to match the changes made, this is awesome work though. 💯

**Review by sylviawanjiku — APPROVED** [review]

**Review by kelvinwm — APPROVED** [review]

**Review by Allan690 — APPROVED** [review]

**Pull request merged**